http/version.go

```diff
@@ -21,6 +21,6 @@ func MakeVersionHandler(componentName, ComponentID, version, environment, gitCom
 			Env:     environment,
 			Commit:  gitCommit,
 		}
-		EncodeReply(context.TODO(), w, info)
+		_ = EncodeReply(context.TODO(), w, info)
```

Comment:
?

Reply:
Reverted to keep sonar code smell and process it correctly when we'll have time